notebook/notebook/notebook.py

```diff
@@ -668,7 +668,7 @@ def notebook_status_from_pod(pod):
                 assert len(workshops) == 0
                 set_message(
                     session,
-                    'No such workshop.  Check the workshop name and password and try again.',
+                    'Workshop Inactive!',
```

Comment:
So I don't like this because if they make a typo during the workshop, the message will be wrong.